behave_django/management/commands/behave.py

```diff
@@ -69,6 +72,18 @@ def add_behave_arguments(parser):  # noqa
         '--simple',
     ]
 
+    def validate_runner_path(path):
```

Comment:
`validate_runner_import` or simply `valid_python_module` (that's what the function is testing, isn't it?), maybe?

Reply:
Done.